image-pipeline/expression-scatter-plots.js

```diff
@@ -360,6 +384,16 @@ async function parseCliArgs() {
   const numCPUs = values.cores ? parseInt(values.cores) : os.cpus().length / 2 - 1
   print(`Number of CPUs to be used on this client: ${numCPUs}`)
 
+
+  // Internal IP address for https://singlecell-staging.broadinstitute.org
+  // Reference: singlecell-01 in
+  // https://console.cloud.google.com/compute/instances?project=broad-singlecellportal-staging
+  // This allows PAPI to access the staging web app server, which is
+  // otherwise blocked per firewall / GCP Cloud Armor.
+  const stagingIP = '10.128.0.5'
```

Comment:
Non-blocking: I think this is fine for PoC work, but really this should be a parameter passed into the PAPI job and set as an environment variable or something similar.

Reply:
Good point, refactored in f6e1698.  See also updated test steps above and [related commands to update Vault](https://broadinstitute.slack.com/archives/G01HY2DJTFY/p1664814286500049).